### ISSUE TITLE
Version mismatch: TFT no version shown

### ISSUE THREAD

**Issue by Neunsen** [Blackymas/NSPanel_HA_Blueprint#1237]
I am trying to upload the TFT and so far I managed to upload the blank.tft without issues.
When I try to upload the eu.tft, the best try was stuck at 53% (which went pretty fast) and then I get this:

WARNING egnspanel.local: Connection error occurred: [Errno 104] Connection reset by peer

I think I did everything correct so far, but this is shown every time:

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/126664391/b2451ded-b9d7-402b-881b-4987625a7052)

How do I resolve this?

**Comment by edwardtfn** [comment]
Please try this:

1. On your ESPHome settings, in the `remote_packages` area, change from `ref: main` to `ref: dev`. This will point to the new upload engine we are developing.
2. Install it in your device (wirelessly is ok).
3. After your panel is back, wait around a minute to give ESPHome a chance to finish the entire boot process.
4. Press "Upload TFT" button. (It would be nice if you can copy the ESPHome logs and share here if it fails).
5. Change it back to `ref: main` and install it again (wirelessly is ok).

**Comment by Neunsen** [comment]
Upload was successful both times. It was smooth and much faster. Thank you. Now the display is showing the loading screen...

**Comment by edwardtfn** [comment]
Which versions are shown for TFT, ESPHome and Blueprint on the blue boot page?

**Comment by Neunsen** [comment]
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/126664391/9b1e9a86-35e3-4433-b9d5-e41e2b1232e1)

**Comment by edwardtfn** [comment]
Are you sure you have flashed it back with `ref: main`?

TFT looks good, so no need to upload it again.

**Comment by Neunsen** [comment]
Did the whole step 5 again. Now its done.

**Comment by edwardtfn** [comment]
So, is the panel working now?

**Comment by Neunsen** [comment]
Yes, I can see the dashboard. Can be closed. Thank you for your quick support.
